cycode/cli/files_collector/sca/base_restore_dependencies.py

```diff
@@ -46,9 +49,11 @@ def try_restore_dependencies(self, document: Document) -> Optional[Document]:
         if self.verify_restore_file_already_exist(restore_file_path):
             restore_file_content = get_file_content(restore_file_path)
         else:
-            restore_file_content = execute_command(
-                self.get_command(manifest_file_path), manifest_file_path, self.command_timeout
+            output_file_path = restore_file_path if self.create_output_file_manually else None
+            execute_command(
```

Comment:
now return of `execute_command` is ignored when create_output_file_manually is False. is it expected?